EMOlater/backend/urls.py

```diff
@@ -1,8 +1,16 @@
 # backend/urls.py
 from django.contrib import admin
 from django.urls import path, include
+from rest_framework import routers
+from .views import *
+
+router = routers.DefaultRouter()
+router.register(r'all_users', UserViewSet)
```

Comment:
is there a reason you're using an `r` string?  There no characters that would merit using one